resources/js/lang/en.js

```diff
@@ -102,17 +93,17 @@ export default {
                 2: "Jekserah has summoned an undead army and is planning to invade the city! You must warn the city guard at once!{br}... or maybe, just maybe, there's something to Jekserah's plan. It makes sense in some dark, twisted sense. You decided not to warn the guards, but instead joined Jekserah's undead army in overthrowing the military."
             },
         },
         12: {
             "name": "The Heart's Desire",
             "stages": {
                 0: "Some of your contacts at the Sanctuary of the Great Oak know of a temple dedicated to an unseen force that will answer any question asked of it. This is a golden opportunity seek out your heart's desire.",
-                1: "Some of your contacts at the Sanctuary of the Great Oak know of a temple dedicated to an unseen force that will answer any question asked of it. This is a golden opportunity seek out your heart's desire.{br}The temple was located high in the Watcher Mountains. The Sanctuary provided you with some basic directions, and you headed off. You risked falling to your death numerous times as you scale the cliff up to the temple. Braving the harsh winds and loose rocks, you finally reached the top and headed through the threshold into a large, open foyer. \"I am afraid I cannot accept visitors at this time. Please come back in twelve years. I may be able to accommodate you then,\" boomed a disembodied voice. Before you could object, a group of massive stone constructs materialized before you. \"I know, I know,\" the voice said. \"You have questions, I have answers. Unfortunately, if you don’t leave now, you won’t have a head to ask the questions in the first place.\" But there was no way you were going to climb back down that cliff empty handed. As the last creature fell, the temple became eerily silent. You called out, but the strange voice offered no response. You began banging the walls with your weapons and yelling even louder, making it clear that you will not leave until you have answers. \"ENOUGH!\" The power of the voice shook the ground. \"I could easily kill you, you know. But normally a threat is enough to send you pathetic lesser beings scrambling. I don’t necessarily like murder, however, so I will make you a deal. I will answer a single question for you, and then you will leave this place without further incident and never return. The voice then offered you a choice of great power, great wealth, or the location of the necromancer.{br}The obvious choice was great power. You are not interested in gold or petty revenge. The only thing that matters is the strength to enact your will. The voice tells you of a shrine in the Copperneck Mountains that contains a source of great power.",
```

Comment:
This didn't even feel like a summary but a complete copy-paste of the previous conclusion. I've mainly removed some sentences containing the minor details.

Reply:
Agreed